AppServices/Validators/CustomerCreateValidator.cs

```diff
@@ -23,7 +22,7 @@ public CustomerValidator()
 
             RuleFor(customer => customer.Cpf)
                 .NotEmpty()
-                .Must(isValidCpf)
+                .Must(ValidatorExtensions.isValidCpf)
                 .WithMessage("Cpf is not valid.");
```

Comment:
Corrigir importação.
```suggestion
                .Must(isValidCpf)
                .WithMessage("Cpf is not valid.");
```